### ISSUE TITLE
Request : Train.py

### ISSUE THREAD

**Issue by Tiantian089** [moralesq/DeepStrain#1]
Dear moralesq,
Hi, could you please share train.py for me to train the model on my own datasets?
Thanks a lot!

Best Wish

**Comment by Hamed-Aghapanah** [comment]
collaborate and write an article base on CMRI  
Dear Manuel A. Morales,
I am hamed Aghapanah , PhD candidate of bio electrics.
I read your  paper about “Deepstrain” and need to apply it on Cardiac MRI with the elastic model.
DeepStrain A Deep Learning Workflow for the Automated Characterization of Cardiac Mechanics
https://www.biorxiv.org/content/10.1101/2021.01.05.425266v1
I build some models to find motion flow but their rmse is not good enough. 
It is related to my PhD thesis and I can participate with you in collaborate and write an article.
shall you help me to do it?
Best Regards,

**Comment by moralesq** [comment]
I will start updating the repo with a clean version of the training code.

**Comment by Hamed-Aghapanah** [comment]
It is a good news.  
Can I help you in this process and then apply your method to my dataset?

**Comment by moralesq** [comment]
Have you been able to use the repo for inference?

**Comment by Hamed-Aghapanah** [comment]
No, I have not worked yet. But I can quickly learn to work with it. 
I search it now.
I can create GUI as inference by pyqt.
